ui/src/pages/ShoeMap.tsx

```diff
@@ -92,6 +80,15 @@ export const ShoeMap: React.FC = () => {
   const currentLocation = { lat: 43.4723, lng: -80.5449 };
   const [isDrawerOpen, setIsDrawerOpen] = useState(StoryDrawerState.Closed);
 
+  const { data:stories, error } = useSWR<Story[]>("/api/stories");
+
+  var markerList = []
+  if(stories && !error){
+    stories.forEach(story => {
+      markerList.push({lat: story.latitude, lng: story.longitude})
```

Comment:
Can we just change the expected markerList type to be stories instead of creating a new array.

You'll have to change the types that are passed down into the `PinCluster` component